Failed build on missing Common.Logging and Quartz

Building the current master of fog-client fails with the following warnings and errors:
```
C:\Program Files\MSBuild\14.0\bin\Microsoft.Common.CurrentVersion.targets(1819,5): warning MSB3245: Could not resolve this reference.
Could not locate the assembly "Common.Logging, Version=3.3.1.0, Culture=neutral, ...

C:\Program Files\MSBuild\14.0\bin\Microsoft.Common.CurrentVersion.targets(1819,5): warning MSB3245: Could not resolve this reference.
Could not locate the assembly "Common.Logging.Core, Version=3.3.1.0, Culture=neutral, ...

C:\Program Files\MSBuild\14.0\bin\Microsoft.Common.CurrentVersion.targets(1819,5): warning MSB3245: Could not resolve this reference.
Could not locate the assembly "Quartz, Version=2.4.1.0, Culture=neutral, ...

PowerManagement\PowerManagement.cs(22,7): error CS0246:
The type or namespace name 'Quartz' could not be found (are you missing a using directive or an assembly reference?) ...
...
```

--- a/Modules/packages.config
+++ b/Modules/packages.config
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Common.Logging" version="3.0.0" targetFramework="net40" />
-  <package id="Common.Logging.Core" version="3.0.0" targetFramework="net40" />
-  <package id="Quartz" version="2.3.3" targetFramework="net40" />
+  <package id="Common.Logging" version="3.3.1.0" targetFramework="net40" />
+  <package id="Common.Logging.Core" version="3.3.1.0" targetFramework="net40" />
+  <package id="Quartz" version="2.4.1" targetFramework="net40" />
   <package id="SharpZipLib" version="0.86.0" targetFramework="net40" />
 </packages>

--- a/README.md
+++ b/README.md
@@ -47,6 +47,8 @@ To checkout and build the entire client (including the Installer) Windows is req
   * Add to PATH - for instructions see below
 * [Windows SDK 7, .NET 4](https://www.microsoft.com/en-us/download/details.aspx?id=8279)
   * Before running the installer **note down** and then adjust some registry keys or the installer will fail (details see [here](https://stackoverflow.com/questions/31455926/windows-sdk-setup-failure))
+    * 32 bit: `HKLM\SOFTWARE\Microsoft\NET Framework Setup\NDP\v4\Client\Version` and `HKLM\SOFTWARE\Microsoft\NET Framework Setup\NDP\v4\Full\Version`
+    * 64 bit: `HKLM\SOFTWARE\Wow6432Node\Microsoft\NET Framework Setup\NDP\v4\Client\Version` and `HKLM\SOFTWARE\Wow6432Node\Microsoft\NET Framework Setup\NDP\v4\Full\Version`
   * Default setup is ok if you have enough disk space left but you can savely go with only selecting '.NET Development' and the two submodules
   * After installation change back the registry keys to what they were before
 
@@ -55,8 +57,8 @@ In case you want to build the Universal Installer you need to add ILMerge as wel
   * (Add to PATH - for instructions see below)
 
 Adding PATH variables:
-* Win+R key -> rundll32.exe sysdm.cpl,EditEnvironmentVariables
-* New... -> Variable name: PATH, Variable value: C:\Program Files\MSBuild\14.0\Bin;C:\Program Files\Microsoft\ILMerge
+* Win+R key -> `rundll32.exe sysdm.cpl,EditEnvironmentVariables`
+* New... -> Variable name: `PATH`, Variable value: `C:\Program Files\MSBuild\14.0\Bin;C:\Program Files\Microsoft\ILMerge`
 
 **Now, restart your system!**
 

--- a/README.md
+++ b/README.md
@@ -34,13 +34,40 @@ Windows      | Linux       | OSX
 
 #### Environment
 
-To build the entire client (including the Installer) Windows is required. This is due to the MSI for network deployment and the Universal Installer. The following dependencies must be installed and included in PATH
+To checkout and build the entire client (including the Installer) Windows is required. This is due to the MSI for network deployment and the Universal Installer. The following dependencies must be installed and included in PATH
+* [Git for Windows](https://git-for-windows.github.io/)
+** All default install settings are fine.
 * [WiX Toolset](http://wixtoolset.org/)
+** Download installer wix???.exe from github repo linked on that website.
 * [Powershell / WMF 4.0+](https://www.microsoft.com/en-us/download/details.aspx?id=40855) **(Windows 7 ships with 3.0)**
+** See Install Instructions on that page to find the correct package for your system
 * [.NET v4.6](https://www.microsoft.com/en-us/download/details.aspx?id=48130)
-* [MSBuild Tools 2015](https://www.microsoft.com/en-us/download/details.aspx?id=48159) **(Included with Visual Studios 2015, add to PATH)**
-* [ILMerge] (https://www.microsoft.com/en-us/download/confirmation.aspx?id=17630) **(Add to PATH)**
-* [Windows SDK 7, .NET 4] (https://www.microsoft.com/en-us/download/details.aspx?id=8279) **(Add to PATH)**
+* [MSBuild Tools 2015](https://www.microsoft.com/en-us/download/details.aspx?id=48159)
+** Included with Visual Studios 2015 if you have that already
+** Add to PATH - for instructions see below
+* [Windows SDK 7, .NET 4] (https://www.microsoft.com/en-us/download/details.aspx?id=8279)
+** Before running the installer **note down** and then adjust some registry keys or the installer will fail (details see [here](https://stackoverflow.com/questions/31455926/windows-sdk-setup-failure))
+** Default setup is ok if you have enough disk space left but you can savely go with only selecting '.NET Development' and the two submodules
+** After installation change back the registry keys to what they were before
+
+In case you want to build the Universal Installer you need to add ILMerge as well:
+* [ILMerge] (https://www.microsoft.com/en-us/download/confirmation.aspx?id=17630)
+** (Add to PATH - for instructions see below)
+
+Adding PATH variables:
+** Win+R key -> rundll32.exe sysdm.cpl,EditEnvironmentVariables
+** New... -> Variable name: PATH, Variable value: C:\Program Files\MSBuild\14.0\Bin;C:\Program Files\Microsoft\ILMerge
+
+**Now, restart your system!**
+
+#### Repository and powershell setting
+Open Git Bash and clone the repository:
+```
+IEUser@WIN7 MINGW32 ~/Desktop
+$ git clone https://github.com/FOGProject/fog-client
+Cloning into 'fog-client'...
+...
+```
 
 Powershell must be configured to allow scripts to be run on the machine. Open CMD as adminstrator and run
 ```

--- a/README.md
+++ b/README.md
@@ -36,7 +36,7 @@ Windows      | Linux       | OSX
 
 To checkout and build the entire client (including the Installer) Windows is required. This is due to the MSI for network deployment and the Universal Installer. The following dependencies must be installed and included in PATH
 * [Git for Windows](https://git-for-windows.github.io/)
-  * All default install settings are fine.
+  * Default install settings are fine.
 * [WiX Toolset](http://wixtoolset.org/)
   * Download installer wix???.exe from github repo linked on that website.
 * [Powershell / WMF 4.0+](https://www.microsoft.com/en-us/download/details.aspx?id=40855) **(Windows 7 ships with 3.0)**
@@ -45,13 +45,13 @@ To checkout and build the entire client (including the Installer) Windows is req
 * [MSBuild Tools 2015](https://www.microsoft.com/en-us/download/details.aspx?id=48159)
   * Included with Visual Studios 2015 if you have that already
   * Add to PATH - for instructions see below
-* [Windows SDK 7, .NET 4] (https://www.microsoft.com/en-us/download/details.aspx?id=8279)
+* [Windows SDK 7, .NET 4](https://www.microsoft.com/en-us/download/details.aspx?id=8279)
   * Before running the installer **note down** and then adjust some registry keys or the installer will fail (details see [here](https://stackoverflow.com/questions/31455926/windows-sdk-setup-failure))
   * Default setup is ok if you have enough disk space left but you can savely go with only selecting '.NET Development' and the two submodules
   * After installation change back the registry keys to what they were before
 
 In case you want to build the Universal Installer you need to add ILMerge as well:
-* [ILMerge] (https://www.microsoft.com/en-us/download/confirmation.aspx?id=17630)
+* [ILMerge](https://www.microsoft.com/en-us/download/confirmation.aspx?id=17630)
   * (Add to PATH - for instructions see below)
 
 Adding PATH variables:

--- a/README.md
+++ b/README.md
@@ -36,27 +36,27 @@ Windows      | Linux       | OSX
 
 To checkout and build the entire client (including the Installer) Windows is required. This is due to the MSI for network deployment and the Universal Installer. The following dependencies must be installed and included in PATH
 * [Git for Windows](https://git-for-windows.github.io/)
-** All default install settings are fine.
+  * All default install settings are fine.
 * [WiX Toolset](http://wixtoolset.org/)
-** Download installer wix???.exe from github repo linked on that website.
+  * Download installer wix???.exe from github repo linked on that website.
 * [Powershell / WMF 4.0+](https://www.microsoft.com/en-us/download/details.aspx?id=40855) **(Windows 7 ships with 3.0)**
-** See Install Instructions on that page to find the correct package for your system
+  * See Install Instructions on that page to find the correct package for your system
 * [.NET v4.6](https://www.microsoft.com/en-us/download/details.aspx?id=48130)
 * [MSBuild Tools 2015](https://www.microsoft.com/en-us/download/details.aspx?id=48159)
-** Included with Visual Studios 2015 if you have that already
-** Add to PATH - for instructions see below
+  * Included with Visual Studios 2015 if you have that already
+  * Add to PATH - for instructions see below
 * [Windows SDK 7, .NET 4] (https://www.microsoft.com/en-us/download/details.aspx?id=8279)
-** Before running the installer **note down** and then adjust some registry keys or the installer will fail (details see [here](https://stackoverflow.com/questions/31455926/windows-sdk-setup-failure))
-** Default setup is ok if you have enough disk space left but you can savely go with only selecting '.NET Development' and the two submodules
-** After installation change back the registry keys to what they were before
+  * Before running the installer **note down** and then adjust some registry keys or the installer will fail (details see [here](https://stackoverflow.com/questions/31455926/windows-sdk-setup-failure))
+  * Default setup is ok if you have enough disk space left but you can savely go with only selecting '.NET Development' and the two submodules
+  * After installation change back the registry keys to what they were before
 
 In case you want to build the Universal Installer you need to add ILMerge as well:
 * [ILMerge] (https://www.microsoft.com/en-us/download/confirmation.aspx?id=17630)
-** (Add to PATH - for instructions see below)
+  * (Add to PATH - for instructions see below)
 
 Adding PATH variables:
-** Win+R key -> rundll32.exe sysdm.cpl,EditEnvironmentVariables
-** New... -> Variable name: PATH, Variable value: C:\Program Files\MSBuild\14.0\Bin;C:\Program Files\Microsoft\ILMerge
+* Win+R key -> rundll32.exe sysdm.cpl,EditEnvironmentVariables
+* New... -> Variable name: PATH, Variable value: C:\Program Files\MSBuild\14.0\Bin;C:\Program Files\Microsoft\ILMerge
 
 **Now, restart your system!**
 

